EXM_125/Makefile

```
@@ -1,62 +1,97 @@
 all: info .setup-cqf-ruler synthea generate-patients calculate-patients
 
+r4: info .setup-cqf-ruler-r4 synthea generate-patients-r4 calculate-patients-r4
+
 info:
 	$(info `make` will cache the cqf-ruler, connectathon, and synythea setups, but repeat patient generation and calculation.)
 	$(info To repeat all steps, do `make clean; make`)
 
 .setup-cqf-ruler: .cqf-ruler connectathon .seed-cqf-ruler
 	touch .setup-cqf-ruler
 
+.setup-cqf-ruler-r4: .cqf-ruler connectathon .seed-cqf-ruler-r4
+	touch .setup-cqf-ruler
```

```suggestion
	touch .setup-cqf-ruler-r4
```

This should match the make target.

---

Good catch. Should be resolved.